### PR TITLE
Adding Org creation and more tests.

### PR DESCRIPTION
**Pull request by joshuaGlass808**
```zsh
±|dev ✔| tasklab
% vendor/bin/phpunit                                                                                                                                                                             joshglass@devbox
PHPUnit 9.3.11 by Sebastian Bergmann and contributors.

.....                                                               5 / 5 (100%)

Time: 00:00.238, Memory: 24.00 MB

OK (5 tests, 8 assertions)

```

### CODE REVIEW

**Comment by joshuaGlass808** [comment]
new test after adding in the "403 test"

```
±|dev ✘| tasklab
% vendor/bin/phpunit                                                                                                                                                                             joshglass@devbox
PHPUnit 9.3.11 by Sebastian Bergmann and contributors.

......                                                              6 / 6 (100%)

Time: 00:00.206, Memory: 24.00 MB

OK (6 tests, 9 assertions)
```

File: app/Http/Controllers/OrganizationController.php [hunk 1]
```diff
@@ -0,0 +1,68 @@
+<?php
```
Comment (jlrusso) [inline app/Http/Controllers/OrganizationController.php]: we gonna declare strict_types on our files?

File: app/Http/Controllers/OrganizationController.php [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+            ], 422);
+        }
+
+        if (Auth::user()->organization_id !== null) {
+            return response()->json([
+                'status' => 'Cannot create new org, you\'re already in an org.'
+            ], 403);
+        }
+
+        DB::transaction(function () use ($request) {
```
Comment (jlrusso) [inline app/Http/Controllers/OrganizationController.php]: : void return type for the function

**Review by jlrusso — APPROVED** [review]
Good stuff. I'll make sure to go test-driven on the front end as well

**Comment by joshuaGlass808** [comment]
@jlrusso I am not too worried about front-end tests at the moment, but if you set it then that's awesome!

**Pull request merged**